### PR TITLE
VB-4: authorization, backend part

### PR DESCRIPTION
**Pull request by dmitriy-uvin**

### CODE REVIEW

File: packages/backend/prisma/seed/data/users.ts [hunk 1]
```diff
@@ -13,10 +12,10 @@ export const users: User[] = [
     role: Role.USER,
     createdAt: new Date('2022-08-09'),
     updatedAt: new Date('2022-08-09'),
+    passwordHash: 'test',
```
Comment (Nightlngale) [inline packages/backend/prisma/seed/data/users.ts:15]: use hashService.generatePasswordHash('test')

File: packages/backend/src/api/routes/auth.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  router.post(
+    apiPath(path, AuthApiRoutes.SIGN_IN),
+    wrap(async (req: Request) => {
+      const signInDto: UserSignInDto = {
+        email: req.body.email,
+        password: req.body.password,
+      };
+
+      return await authService.signIn(signInDto);
```
Comment (Nightlngale) [inline packages/backend/src/api/routes/auth.ts:25]: You don't need to await Promise unless you will use its result

File: packages/backend/src/error/error.handler.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  }
+
+  if (err instanceof HttpError) {
+    return res.status(err.status).json({
+      error: err.message,
+    });
+  }
+
+  return res.status(HttpStatusCode.INTERNAL_SERVER_ERROR).json({
+    error: 'interna;',
```
Comment (Nightlngale) [inline packages/backend/src/error/error.handler.ts:21]: internal?

File: packages/backend/src/api/routes/auth.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+export const initAuthRoutes = (
+  { authService }: Services,
+  path: ApiRoutes,
+): Router => {
+  const router = Router();
+
+  router.post(
+    apiPath(path, AuthApiRoutes.SIGN_IN),
+    wrap(async (req: Request) => {
```
Comment (Nightlngale) [inline packages/backend/src/api/routes/auth.ts:19]: here you can use `wrap<Empty, TokensDto, UserSignInDto>`, and then just use 

`return authService.signIn(req.body);`

File: packages/backend/src/api/routes/auth.ts [hunk 5]
```diff
@@ -0,0 +1,68 @@
+import type { Services } from '@services';
+import { type Request, Router } from 'express';
+import type { ApiRoutes, UserSignInDto, UserSignUpDto } from '@vse-bude/shared';
+import { AuthApiRoutes } from '@vse-bude/shared';
+import { wrap } from '@helpers';
+import { apiPath } from '../../helpers/api';
```
Comment (Nightlngale) [inline packages/backend/src/api/routes/auth.ts:6]: use import aliases and export from index.

File: packages/backend/src/api/routes/auth.ts [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+import { authMiddleware } from '../../auth/middlewares/auth.middleware';
+import type { SignOutDto } from '../../common/types/auth/sign-out.dto';
+import type { UpdateRefreshTokenDto } from '../../common/types/refresh-token/update.refresh-token.dto';
```
Comment (Nightlngale) [inline packages/backend/src/api/routes/auth.ts:9]: use kebab-case for file names and don't add type after dot
 
 just update-refresh-token

**Review by Nightlngale — CHANGES_REQUESTED** [review]

**Comment by dmitriy-uvin** [comment]
@Nightlngale fixed

**Review by Nightlngale — APPROVED** [review]

**Pull request merged**